src/coffee/fetchstocks.coffee

```diff
@@ -8,7 +8,8 @@ class FetchStocks
 
 
   run: ->
-    query = @client.inventoryEntries.all().sort('sku').expand('supplyChannel')
+    # Do NOT use custom sorting due to https://github.com/sphereio/sphere-node-sdk/issues/194
```

Comment:
Think I agree with Abi here. If in the future somebody thinks it's a good idea to sort by SKU your test will fail. Then he/she will find out that way. Or am I missing something?